traefik/values.yaml

```diff
@@ -57,6 +57,21 @@ deployment:
   # Additional imagePullSecrets
   imagePullSecrets: []
     # - name: myRegistryKeySecretName
+  # Use readiness gates (e.g to specify a list of additional conditions for pod readiness)
+  readinessGates: []
+    # Example with AWS LB Controller to set pod healthiness based on its registration state on a given Application Load Balancer
+    # - conditionType: target-health.alb.ingress.k8s.aws/traefik-ingress_traefik-service_9000
```

Comment:
This is legacy configuration and shouldn't be recommended as example

Reply:
Thank you @z0rc for keeping an eye on that. Do you know what is the correct example for the readinessgate? If so, can you please share it with us? Thank you,

Reply:
I don't have a good example. AWS LB Controller injects readinessgate on the flight, gate names are also generated and don't make much sense to user. I'm not aware of other examples where and how user-defined readinessgates can/should be used.

Reply:
You are right about the readiness gate (as it can be done by adding a label to the namespace) but we still need the lifecycle properties to let the pod be in terminated state long enough so that the ALB can drain it

Reply:
basically the idea is that the pod can still serve traffic even if it is technically terminated. this helps to remove it from target group without causing outage

Reply:
```yml
apiVersion: v1
kind: Namespace
metadata:
  labels:
    elbv2.k8s.aws/pod-readiness-gate-inject: enabled
  name: traefik
```

Reply:
https://github.com/kubernetes-sigs/aws-load-balancer-controller/issues/1064 reference for the need of preStop hook